redis_json/src/commands.rs

```diff
@@ -588,17 +594,32 @@ pub fn json_mget<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>)
 
     args.last().ok_or(RedisError::WrongArity).and_then(|path| {
         let path = Path::new(path.try_as_str()?);
-        let keys = &args[1..args.len() - 1];
 
-        let format_options = FormatOptions {
-            resp3: is_resp3(ctx),
-            ..Default::default()
+        let is_resp3 = is_resp3(ctx);
+        let mut format_options = FormatOptions::new(is_resp3);
+
+        // Check if the optional argument [FORMAT STRING|JSON|EXPAND] is present
+        let keys = if is_resp3 {
+            let format = &args[args.len() - 3..args.len() - 1];
```

Comment:
Wouldn't this raise a panic if out of range?

Reply:
it will never fall since we check that the command has more than 3 arguments